src/main/java/io/temporal/samples/dsl/models/ActivityInvocation.java

```diff
@@ -0,0 +1,54 @@
+package io.temporal.samples.dsl.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Strings;
+import io.temporal.activity.ActivityOptions;
+import io.temporal.workflow.ActivityStub;
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import java.util.Map;
+
+public class ActivityInvocation {
+  public String name;
+  public String[] arguments;
+  public String result;
+
+  @JsonCreator
+  public ActivityInvocation(
+      @JsonProperty("name") String name,
+      @JsonProperty("arguments") String[] arguments,
+      @JsonProperty("result") String result) {
+    this.name = name;
+    this.arguments = arguments;
+    this.result = result;
+  }
+
+  public Void execute(Map<String, String> bindings) {
+    String[] args = makeInput(this.arguments, bindings);
+    // Class.forName(name);
+
+    ActivityStub stub =
+        Workflow.newUntypedActivityStub(
+            ActivityOptions.newBuilder()
+                .setStartToCloseTimeout(Duration.ofMinutes(1))
+                .setTaskQueue("dsl")
+                .build());
+
+    String results = stub.execute(name, String.class, new Object[] {args});
```

Comment:
It never picks up to execute the activity.  It register the event thou
![image](https://user-images.githubusercontent.com/16171726/94345100-7211a800-ffd8-11ea-9050-0332e18d3324.png)

![image](https://user-images.githubusercontent.com/16171726/94345106-7fc72d80-ffd8-11ea-9c0a-00e673af754f.png)

![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/16171726/94345440-9e2e2880-ffda-11ea-841e-4b2c00619598.png)

So i found out the issue. There two problems
1) It's looking for the method name of the activity (entry point) vs. the activity itself
2) in java functions are lowercase, but in this case it requires the `GetInfo` vs. `getInfo`

Reply:
So the issue 1 make sense since you need to specify what method to call in the activity.  I guess i need to figure out what the code would be to generate an activity stub based on activity name...and then use the stub to call the activity method, both of which needs to be in the yaml (activity, method name)

Reply:
@mfateev what would the code look like if i wanted to trigger it based on the activity name + the activities's method name.  Stub.execute("activityname":"method_name")? MethodName captalized?